src/systemtask/SystemTask.cpp

```diff
@@ -81,6 +81,7 @@ SystemTask::SystemTask(Drivers::SpiMaster& spi,
     dateTimeController {dateTimeController},
     timerController {timerController},
     alarmController {alarmController},
+    alertController(motorController),
```

Comment:
It's not possible to initialize `alertController` of type `Pinetime::Controllers::AlertController` with `motorController` of type  `Pinetime::Controllers::MotorController` since those are two different types.